### PR TITLE
Making changes to ALARA flux file and input

### PR DESCRIPTION
**Pull request by anu1217**
- Changed flux values to go from high to low energy in W wall
- Increased tolerance to gather data for more nuclides

### CODE REVIEW

File: PointSourceALARA [hunk 1]
```diff
@@ -47,4 +47,4 @@ output zone
    number_density
 end
 
-truncation 1e-6
+truncation 5e-15
```
Comment (gonuke) [inline PointSourceALARA:50]: This is much lower than I would typically expect to see - do you need this to get results similar to OpenMC?

File: SS_Activation_OpenMC.py [hunk 2]
```diff
@@ -84,9 +84,9 @@
 #Depletion calculation
 W.depletable = True
 W.volume = 4.0/3.0 * np.pi * (R_2**3 - R_1**3) #volume of W wall material
-fluxes, micros = openmc.deplete.get_microxs_and_flux(model, Cells)
-operator = openmc.deplete.IndependentOperator(materials, fluxes[0:1], micros[0:1],normalization_mode='source-rate')
-# operator = openmc.deplete.CoupledOperator(model, normalization_mode='source-rate')
+#fluxes, micros = openmc.deplete.get_microxs_and_flux(model, Cells)
+#operator = openmc.deplete.IndependentOperator(materials, fluxes[0:1], micros[0:1],normalization_mode='source-rate')
```
Comment (gonuke) [inline SS_Activation_OpenMC.py:88]: Let's delete lines we don't need instead of commenting them out

File: SS_Activation_OpenMC.py [hunk 3]
```diff
@@ -127,25 +127,29 @@
 # Stable W nuclides present at beginning of operation (will not be plotted)
 stable_nuc = ['W180', 'W182', 'W183', 'W184', 'W186']  
 
-#Store list of nuclides from last timestep as a Materials object
-materials_last = results.export_to_materials(-1)
-# Storing depletion data from 1st material
-mat_dep = materials_last[0]
-# Obtaining the list of nuclides from the results file
-nuc_last = mat_dep.get_nuclides()
+materials_list=[]
```
Comment (gonuke) [inline SS_Activation_OpenMC.py:130]: These changes don't seem related to this PR.  Maybe you branched from another branch, but if you are only changing the ALARA inputs in ways that don't depend on these changes, you should branch from main and make a self-contained PR for just those things.

File: PointSourceALARA (hunk 1 above)
Comment (gonuke) [reply PointSourceALARA:50]: Have you tried something in-between?  Does OpenMC have a threshold?  Such a low threshold can become problematic in some scenarios.

**Review by gonuke — CHANGES_REQUESTED** [review]
Thanks for the updates here.  It's not clear why you have a new flux file?  Why did it change?

File: PointSourceALARA (hunk 1 above)
Comment (anu1217) [reply PointSourceALARA:50]: With 1e-6 ALARA was making too few nuclides and there wasn't much to compare with OpenMC. I can try something in between the two -- 1e-15 was as low as it would go without getting stuck

**Comment by anu1217** [comment]
I noticed the Neutron_Flux.csv file was still going from low to high energy, even though the OpenMC script has it written from high to low. The flux file currently in the repo may be from an older version of the OpenMC script
